server/index.js

```diff
@@ -6,15 +6,21 @@ var app = express();
 var bodyParser = require('body-parser');
 var title = require('./routes/title');
 var enrolled = require('./routes/enrolled');
+const mongoose = require('mongoose');
 
-  app.use(express.static(__dirname + '/../client/dist'));
+app.use(express.static(__dirname + '/../client/dist'));
 app.use(cors());
```

Comment:
I like it!  I was going to suggest using cors();  You beat me to it.

Reply:
Life saver this one little line